incubator/hnc/README.md

```diff
@@ -69,7 +74,7 @@ I do most of my testing in [KIND](https://kind.sigs.k8s.io).
 
 ### Testing on KIND (Kubernetes IN Docker)
 
-* Run `. devenv` to setup your `KUBECONFIG` and `PATH` env vars correctly.
+* Run `source devenv` to setup your `KUBECONFIG` and `PATH` env vars correctly.
```

Comment:
Why this change?

Reply:
The previous command (`. dotenv`) didn't work for me, but `source dotenv`. I think I'll add this as another option rather than removing the first one.

Reply:
Ah, what shell are you using? I think `.` only works in bash.

Reply:
I use zsh :upside_down_face: